tests/integration-tests/src/VuFindTest/OaiPmh/HarvesterFactoryTest.php

```diff
@@ -168,91 +168,6 @@ public function testSSLVerifyPeer()
         $this->getHarvester('test', sys_get_temp_dir(), $config, $client);
     }
 
-    /**
-     * Test date autodetection.
-     *
-     * @return void
-     */
-    public function testDateAutodetect()
```

Comment:
Within the context of HarvesterFactoryTest I don't think it's relevant to be testing any Identify calls or granularity detection anymore. This is because getIdentifyResponse() now allows Identify calls and granularity detection to be done lazily, instead of running as part of Harvester instantiation. I think that justifies the removal of these 3 related tests in HarvesterFactoryTest. Other adjustments in HarvesterTest continue some coverage for granularity check.